packages/components/src/Select.js

```diff
@@ -34,12 +34,16 @@ export const Select = React.forwardRef((props, ref) => (
         bg: 'transparent',
       }}
     />
-    <DownArrow
-      sx={{
-        ml: -28,
-        alignSelf: 'center',
-        pointerEvents: 'none',
-      }}
-    />
+    {arrow ? (
+      arrow
+    ) : (
```

Comment:
```suggestion
    {arrow || (
```

Reply:
@lachlanjc  thanks a lot for reviewing - I checked in changes with the simplified syntax